core/src/main/resources/lib/hudson/project/configurable.jelly

```diff
@@ -30,7 +30,10 @@ THE SOFTWARE.
         <l:task href="${url}/build?delay=0sec" icon="icon-clock icon-md" onclick="${it.parameterized?null:'return build_' + id + '(this)'}" permission="${it.BUILD}" post="${!it.parameterized}" title="${it.buildNowText}"/>
         <script>
             function build_${id}(a) {
-                new Ajax.Request(a.href);
+                fetch(a.href, {
+                  method: 'post',
+                    headers: crumb.wrap({}),
```

Comment:
Does this need linting? (or is the additional spacing expected)

Reply:
```suggestion
                  headers: crumb.wrap({}),
```

Reply:
There's no linting done on jelly files, but fixed it here.